zppy/templates/tc_analysis.bash

```diff
@@ -90,13 +90,6 @@ cat ${result_dir}out.dat0* > ${result_dir}cyclones_${file_name}.txt
 StitchNodes --in_fmt "lon,lat,slp,wind" --in_connect ${result_dir}connect_CSne${res}_v2.dat --range 6.0 --mintime 6 --maxgap 1 --in ${result_dir}cyclones_${file_name}.txt --out ${result_dir}cyclones_stitch_${file_name}.dat --threshold "wind,>=,17.5,6;lat,<=,40.0,6;lat,>=,-40.0,6"
 rm ${result_dir}cyclones_${file_name}.txt
 
-# If cyclones_stitch file is empty, exit
-if ! [ -s ${result_dir}cyclones_stitch_${file_name}.dat ]; then
-  cd {{ scriptDir }}
-  echo 'ERROR (1)' > {{ prefix }}.status
-  exit 1
-fi
-
```

Comment:
Key Change 3: Removed this block since this is not an error per-se, as discussed. Instead, E3SM Diags will do this check via https://github.com/E3SM-Project/e3sm_diags/commit/ffbdf287a91eca57fd383a1de4c0cf22e9fce829